requirements.txt

```diff
@@ -4,3 +4,4 @@ south
 whoosh
 ipy
 python-ldap
+djangorestframework==2.4.1
```

Comment:
While trying to install I faced various problems with the djangorestframework I was trying to install. I cannot actually recall them, but I pinned the version as a precaution e.g. I remember v2.4 was not working.